app/ts/utils/utils.ts

```diff
@@ -0,0 +1,17 @@
+export const bigintSecondsToDate = (seconds: bigint) => {
+	if (seconds > Number.MAX_SAFE_INTEGER) throw new Error(`Too big seconds value: ${ seconds }`)
```

Comment:
Same comment as that other PR.  This should not be `Number.MAX_SAFE_INTEGER`, it should be whatever that number I found on Stack Overflow is for the largest number of seconds a `Date` can be.